src/scripts/supported-images/versions.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * Creates an HTML list that can be used to document 'Supported' and 'Deprecated' image
+ * versions in language and service pages pulled from 'images.json'. To include use:
+ *
+ * To list the supported versions of the service InfluxDB:
+ *
+ * <div id = "influxdbSupported"></div>
+ *
+ * <script>
+ * makeImagesList("services", "influxdb", "supported", "influxdbSupported");
+ * </script>
+ *
+ * @param str imageType Dictates whether list will be made for and image from "runtimes" or "services".
+ * @param str childImage The individual image the list is made from (i.e., "php", "nodejs", "mongodb", "varnish").
+ * @param str attribute Which attribute of that image JSON (containing a list) will be listed ("supported", "deprecated").
+ * @param str divName References the id of <div> in the Markdown document.
+ *
+ */
+function makeImagesList(imageType, childImage, attribute, divName) {
```

Comment:
We can likely clean this up considerably, once we decide if we want to do it at all. If so, it may be a good time to consider a micro-templating approach courtesy of ES6, with or without something like HTM: https://github.com/developit/htm